main.nf

```diff
@@ -18,46 +18,54 @@ def helpMessage() {
 
     The typical command for running the pipeline is as follows:
 
-    nextflow run nf-core/viralrecon --input samplesheet.csv --viral_genome 'NC_045512.2' -profile docker
+    nextflow run nf-core/viralrecon --input samplesheet.csv --host_genome 'human' --viral_genome 'NC_045512.2' -profile docker
 
     Mandatory arguments
       --input [file]                  Comma-separated file containing information about the samples in the experiment (see docs/usage.md)
-      --viral_fasta [file]            Path to Fasta reference for viral genome
       -profile [str]                  Configuration profile to use. Can use multiple (comma separated)
                                       Available: conda, docker, singularity, test, awsbatch, <institute> and more
 
     Generic
       --protocol [str]                Specifies the type of protocol used for sequencing i.e. "metagenomic" or "amplicon". (Default: "metagenomic")
-      --amplicon_fasta                Path to Fasta file containing amplicon sequences
+      --amplicon_fasta [file]         Path to fasta file containing amplicon sequences
 
     References                        If not specified in the configuration file or you wish to overwrite any of the references
-      --host_kraken2_db [file]        Full path to Kraken2 database built from host genome
-      --host_kraken2_name [str]       Name of host genome for building Kraken2 database (Default: "human")
+      --host_genome [str]             Name of genome reference key for human genome
 
-      --viral_genome [str]            Name of iGenomes reference for viral genome
-      --viral_kraken2_db [file]       Full path to Kraken2 database built from viral genome
+      --viral_genome [str]            Name of genome reference key for viral genome
```

Comment:
What is a reference key exactly?  Might be useful to have a default or example value in the description here

Reply:
Dont quite know yet @heuermh because I am still figuring out how best to organise the data by genome...add placeholders in the commits below:

See https://github.com/nf-core/viralrecon/pull/21/commits/a0a0cab43fadc18b516a6f46a159ab2f997fd1ad and https://github.com/nf-core/viralrecon/pull/21/commits/445f79ef5635af1f26485d813e10cd6496a55a2a